src/script/lua_api/l_object.cpp

```diff
@@ -1023,22 +1023,54 @@ int ObjectRef::l_get_look_dir(lua_State *L)
 	return 1;
 }
 
+// DEPRECATED
 // get_look_pitch(self)
 int ObjectRef::l_get_look_pitch(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
+
+	log_deprecated(L, "Deprecated call to get_look_pitch instead of get_look_pitch2");
+	
 	ObjectRef *ref = checkobject(L, 1);
 	Player *player = getplayer(ref);
 	if (player == NULL) return 0;
 	// Do it
-	lua_pushnumber(L, player->getRadPitch());
+	lua_pushnumber(L, player->getRadPitchDep());
 	return 1;
 }
 
+// DEPRECATED
 // get_look_yaw(self)
 int ObjectRef::l_get_look_yaw(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
+
+	log_deprecated(L, "Deprecated call to get_look_yaw instead of get_look_yaw2");
```

Comment:
It should be made more obvious which of them is the deprecated one and which isn't. Maybe replace it with  "Deprecated call to get_look_yaw, use get_look_yaw2 instead"?